api/users/pb/users.proto

```diff
@@ -19,6 +19,7 @@ message GetThreadRequest {
 message GetThreadReply {
     bytes ID = 1;
     string name = 2;
+    bool isDB = 3;
```

Comment:
This is needed to differentiate between thread IDs for DBs, which can be used for buckets, or raw / log type thread IDs.